### PR TITLE
feat: add BackTop duration

### LINKED ISSUE #25251: 希望 BackTop 能增加一个duration属性
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
实现自定义弹回顶部所需时间

### What does the proposed API look like?
<BackTop duration={1000}/>

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
fix #25251

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Add BackTop `duration` used to set the time required to return to the top.   |
| 🇨🇳 Chinese |  增加 BackTop `duration` 用来设置回到顶部所需时间。         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/back-top/index.tsx [hunk 1]
```diff
@@ -68,9 +69,10 @@ const BackTop: React.FC<BackTopProps> = props => {
   };
 
   const scrollToTop = (e: React.MouseEvent<HTMLDivElement>) => {
-    const { onClick, target } = props;
+    const { onClick, target, duration = 450 } = props;
     scrollTo(0, {
```
Comment (xrkffgg) [inline components/back-top/index.tsx:73]: 这个 0 要不要也提出来，可以设置 回到的位置

**Review by afc163 — APPROVED** [review]

**Comment by xrkffgg** [comment]
这个的 文档 API 我 补到 feature 还是 master ？ @afc163

**Comment by afc163** [comment]
feature 吧

**Pull request merged**